### PR TITLE
fix: v2 seated bookings

### PR DESCRIPTION
**Pull request by supalarry**
Linear [CAL-4438](https://linear.app/calcom/issue/CAL-4438/platform-fix-v2-seated-bookings)

### CODE REVIEW

File: apps/api/v2/src/ee/bookings/2024-08-13/bookings.repository.ts [hunk 1]
```diff
@@ -29,6 +29,25 @@ export class BookingsRepository_2024_08_13 {
...
+          in: ids,
+        },
+      },
+      include: {
+        attendees: {
+          include: {
+            bookingSeat: true,
+          },
+        },
+        user: true,
```
Comment (ThyMinimalDev) [inline apps/api/v2/src/ee/bookings/2024-08-13/bookings.repository.ts:45]: are we formatting the password out of the response data ?

Comment (ThyMinimalDev) [reply apps/api/v2/src/ee/bookings/2024-08-13/bookings.repository.ts:45]: seems to be done in output service 👍

**Review by ThyMinimalDev — APPROVED** [review]
HQ pr

File: apps/api/v2/src/ee/bookings/2024-08-13/bookings.repository.ts (hunk 1 above)
Comment (supalarry) [reply apps/api/v2/src/ee/bookings/2024-08-13/bookings.repository.ts:45]: Yes - we have `@Expose()` and in output service we use `plainToClass` with corresponding class.

**Pull request merged**